include/LightGBM/config.h

```diff
@@ -27,7 +27,7 @@ namespace LightGBM {
 
 /*! \brief Types of tasks */
 enum TaskType {
-  kTrain, kPredict, kConvertModel, KRefitTree
+  kTrain, kPredict, kConvertModel, KRefitTree, kSaveBinary
```

Comment:
@cyfdecyf Thanks for your contribution! Could you please resolve all conflicts and document new enum option?
https://github.com/microsoft/LightGBM/blob/ed651e86729477e7d2699534d63a5cace87f7878/include/LightGBM/config.h#L95-L106
After adding description of `save_binary` task, run this script to regenerate documentation.
https://github.com/microsoft/LightGBM/blob/master/helpers/parameter_generator.py

Reply:
Sorry for the late reply. Conflicts are resolved and added document for the new enum option.